docs/modules/operation/pages/deep-dive/admin/webui/introduction.adoc

```diff
@@ -1,4 +1,63 @@
 
-= Administrative Web Interface
+= Admin UI Customization
 
-This section describes the main sections of the {page-component-title} web UI.
\ No newline at end of file
+This section describes how to customize the main components of the {page-component-title} web UI.
+
+== Dashboard
+
+A dashboard is an at-a-glance overview of your monitored network; it can be used as a starting point for more detailed diagnoses of issues.
+You can customize your dashboard using its predefined components.
+
+For more information, see xref:deep-dive/admin/webui/dashboard.adoc[].
```

Comment:
Dashboard is general Web UI info, not specific to the "Admin UI" section.  The "Surveillance View" section is how to configure the Dashboard, but this section is referencing things outside the Admin UI